server/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "lsp-prisma-server",
+    "description": "Implementation of a language server in node.",
+    "version": "0.0.30",
+    "author": "Prisma",
+    "license": "Apache-2.0",
+    "engines": {
+        "node": "*"
+    },
+    "prisma": {
+        "version": "76857c35ba1e1764dd5473656ecbbb2f739e1822"
+    },
+    "repository": {
+        "type": "github",
+        "url": "https://github.com/prisma/vscode"
+    },
+    "dependencies": {
+        "@prisma/fetch-engine": "^2.0.0-beta.2",
+        "@prisma/get-platform": "0.1.x",
+        "@prisma/sdk": "2.0.0-beta.2",
```

Comment:
Hm, this seems like something that needs to be taken into consideration of the update script by @divyenduz as well.

Reply:
Ah! the update script must bump these too. But what is @prisma/get-platform and why is its version not in sync? 

Asking the CLI channel.

Reply:
Ah! looks like its version should match too. A change for @carmenberndt 

![image](https://user-images.githubusercontent.com/746482/79865074-34d93680-83f8-11ea-9553-5e868689da66.png)